jwst/cube_build/ifu_cube.py

```diff
@@ -1339,23 +1340,13 @@ def setup_ifucube_wcs(self):
         final_b_max = max(corner_b)
 
         log.debug(f'final a and b:{final_a_min, final_b_min, final_a_max, final_b_max}')
-        log.debug(f'final wave:   {min(lambda_min), max(lambda_max)}')
-
-        # for MIRI wavelength range read in from cube pars reference file
-        if self.instrument == 'MIRI':
-            final_lambda_min = self.wavemin
-            final_lambda_max = self.wavemax
-
-        # for NIRSPec wavelength range determined from the data
-        if self.instrument == 'NIRSPEC':
-            final_lambda_min = min(lambda_min)
-            final_lambda_max = max(lambda_max)
-            if self.wavemin_user:
-                final_lambda_min = self.wavemin
-
-            if self.wavemax_user:
-                final_lambda_max = self.wavemax
+        log.debug(f'final wave using data:   {min(lambda_min), max(lambda_max)}')
```

Comment:
If the wavelength limits are now determined from the cubepars ref file for both MRS and NRS-IFU, instead of using the data itself, should this log message be updated, so that it doesn't say "using data"?